tests/integration/profiles/five-safes-crate/test_5src_10_outputs.py

```diff
@@ -22,32 +22,30 @@
 logger = logging.getLogger(__name__)
 
 
-# ----- MUST fails tests
+# ----- SHOULD fails tests
 
 
-def test_result_output_does_not_exist():
+def test_completed_createaction_does_not_have_result():
     """
-    Test a Five Safes Crate where an output listed in
-    `CreateAction` --> `result` does not exist as an
-    entity in the metadata file.
-    (We remove the triplets with an output as the subjects)
+    Test a Five Safes Crate where `CreateAction` has `CompleteActionStatus` but
+    does not have the property `result`.
+    (We remove `result` from `CreateAction` if this has `CompleteActionStatus`)
     """
     sparql = (
         SPARQL_PREFIXES
         + """
         DELETE {
-            ?output ?p ?o .
+            ?action schema:result ?o .
         }
         WHERE {
-            ?action schema:result ?output ;
-                    a schema:CreateAction .
-        }
+            ?action a schema:CreateAction ;
+                      schema:actionStatus "http://schema.org/CompleteActionStatus" .        }
```

Comment:
Just spotted this - needs to be changed throughout (http://schema.org/CompleteActionStatus doesn't resolve, http://schema.org/CompletedActionStatus does)

```suggestion
                      schema:actionStatus "http://schema.org/CompletedActionStatus" .        }
```

Reply:
Done.